Dockerfile

```
@@ -54,7 +54,7 @@ RUN \
 
 # Install GHC.
 
-ARG GHC_VERSION=9.0.2
+ARG GHC_VERSION=9.4.1
```

```suggestion
ARG GHC_VERSION=9.0.2
```
I can't upgrade this by default yet because HLS doesn't support 9.4.1. I could update the default to not have HLS, but I feel like 9.0.2 with HLS is a better default.